snap/local/custom-wrapper

```diff
@@ -3,4 +3,6 @@
 export XDG_CONFIG_HOME="$SNAP_REAL_HOME/.config"
 export XDG_DATA_HOME="$SNAP_REAL_HOME/.local/share"
 
+export XCURSOR_PATH="/usr/share/icons"
```

Comment:
Two suggestions for this override. Both of them also resolve the cursor theme issues :slightly_smiling_face:

1. Include existing `XCURSOR_PATH` paths set by `snapcraft-desktop-helpers`.

```suggestion
export XCURSOR_PATH="/usr/share/icons:$XCURSOR_PATH"
```

2. Completely unset `XCURSOR_PATH` overrides, and leave it to the system defaults.

```suggestion
unset XCURSOR_PATH
```